openedx/core/djangoapps/catalog/tests/factories.py

```diff
@@ -88,6 +88,7 @@ class CourseRunFactory(DictFactoryBase):
     image = ImageFactory()
     key = factory.LazyFunction(generate_course_run_key)
     marketing_url = factory.Faker('url')
+    eligible_for_financial_aid = True
```

Comment:
I think this is necessary given the fact that CourseRunFactory [here](https://github.com/edx/edx-platform/pull/14739/files/013348daa1d7846dedc565ad41b95df169b7d0f7#diff-b5964abcd4d8c0c5b574d6df3aaa312fR84) is not a model based factory.